eslint.config.js

```diff
@@ -6,7 +6,7 @@ export default [
       mocha
     },
     languageOptions: {
-      ecmaVersion: 8,
+      ecmaVersion: "latest",
```

Comment:
Was this intentional? I'm never sure what one is _supposed_ to do here :-)

Reply:
Ah right, `import.meta.url` should be `ecmaVersion: 11`? (2020) Perhaps better not to use latest since it's a bit of a moving target?

Reply:
I tried setting 11 and it says

```
TypeError: Key "languageOptions": Key "ecmaVersion": Expected a number or "latest".
```

And it looks like it expects specific values, but they do recommend setting it as "latest" at
TypeError: Key "languageOptions": Key "ecmaVersion": Expected a number or "latest".

Reply:
Did you try`: "11"` or `: 11` - needs to be the latter. :)

Linting against the latest version seems great if you have no specific requirements outside your app for support of that `ecmaVersion` or otherwise have the babel magic to get where you need to go, but otherwise I'd have thought it could mean allowing (not not failing on) new syntax that is not actually implemented by your given minimum support runtime. Given how slow things are to get into stage 3/4 proposals, and how fast Node is to implement them, it's possibly moot.

Your call.

Reply:
🤦‍♂️ I tried `"11"` pushed commit a5b267043c09